.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
```

Comment:
Would you please migrate to GitHub actions as the use of CircleCI is depreacted at Mattermost? You can use https://github.com/mattermost/mattermost-app-google-drive/blob/master/.github/workflows/ci.yml as an example.

Reply:
migrated